engine/admin/prometheus.md

```diff
@@ -0,0 +1,101 @@
+---
+description: Collecting Docker metrics with Prometheus
+keywords: prometheus, metrics
+title: Collect Docker metrics with Prometheus
+---
+
+[Prometheus](https://prometheus.io/) is an open-source systems monitoring and
+alerting toolkit. You can configure Docker as a Prometheus target. This topic
+shows you how to configure Docker, set up Prometheus to run as a Docker
+container, and monitor Docker using Prometheus.
+
+> **Warning**: The available metrics and the names of those metrics are in
+> active development and may change at any time.
+
+## Configure Docker
+
+To configure the Docker daemon as a Prometheus target, you need to specify the
+`metrics-address`. The best way to do this is via the `daemon.json`, which is
+located at one of the following locations by default. If the file does not
+exist, create it.
+
+- **Linux**: `/etc/docker/daemon.json`
+- **Windows Server**: `C:\ProgramData\docker\config\daemon.json`
+- **Docker for Mac / Docker for Windows**: Click the Docker icon in the toolbar,
+  select **Preferences**, then select **Daemon**. Click **Advanced**.
+
+If the file is currently empty, paste the following:
+
+```json
+{
+  "metrics-addr" : "127.0.0.1:9323",
+  "experimental" : true
+}
+```
+
+If the file is not empty, add those two keys, making sure that the resulting
+file is valid JSON. Be careful that every line ends with a comma (`,`) except
+for the last line.
+
+Save the file, or in the case of Docker for Mac or Docker for Windows, save the
+configuration. Restart Docker.
+
+Docker now exposes Prometheus-compatible metrics on port 9323.
+
+## Configure and run Prometheus
+
+In this example, Prometheus runs as a Docker container on the same host.
+
+1.  Copy the following and save it to `/tmp/prometheus.yml`. This is a stock Prometheus
+    configuration file, except for the addition of the Docker daemon as a target,
+    in the last line of the file.
+
+    ```yml
+    # my global config
+    global:
+      scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+      evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+      # scrape_timeout is set to the global default (10s).
+
+      # Attach these labels to any time series or alerts when communicating with
+      # external systems (federation, remote storage, Alertmanager).
+      external_labels:
+          monitor: 'codelab-monitor'
+
+    # Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+    rule_files:
+      # - "first.rules"
+      # - "second.rules"
+
+    # A scrape configuration containing exactly one endpoint to scrape:
+    # Here it's Prometheus itself.
+    scrape_configs:
+      # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+      - job_name: 'prometheus'
```

Comment:
You should create a new job for `docker`. This job is for monitoring prometheus itself. Just add another section like this:

```
- job_name: 'docker'
         # metrics_path defaults to '/metrics'
         # scheme defaults to 'http'.
 
         static_configs:
           - targets: ['localhost:9323']
```